build/symlink-dependencies/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf').sync;
+
+const Project = require('../utils/project');
+
+/**
+ * Ember CLI addon that symlinks cross-package dependencies. This makes it much
+ * easier to test and `npm link` packages locally before they are published.
+ *
+ * This is implemented using an addon because there's no hook in Broccoli itself
+ * to run some code *after* a build has finished. Additionally, Ember CLI uses
+ * tree-sync to efficiently copy the output from Broccoli's tmp directory to
+ * dist. Unfortunately, these optimizations are incompatible with symlinks (it
+ * can't handle circular links) so the symlinking must happen after the build
+ * process has finished.
+ */
+module.exports = {
+  name: 'symlink-dependencies',
+
+  preBuild() {
+    rimraf('dist/@glimmer/*/node_modules');
```

Comment:
Is line 29 the best way to determine target output directory for the current build?